app/display.c

```diff
@@ -159,10 +171,80 @@ void display_init(void)
     scroll_message_row = ROW_SCROLL_T;
 }
 
+void display_cpu_topology(void)
+{
+    extern int num_enabled_cpus;
+    int num_cpu_sockets = 1;
+
+    if(smp_enabled) {
```

Comment:
Would prefer space between "if" and "(" (throughout this function). The rationale is this makes "if" statements visually distinct from function calls.

Reply:
Fixed (spaces added)